styles.css

```diff
@@ -1 +1,212 @@
-body
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Merriweather&family=Roboto&display=swap');
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+* {
+    margin: 0;
+    box-sizing: border-box;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+body {
+    font-family: "Roboto", Arial, sans-serif;
+    font-size: 24px;
+    color: #0C0C0C;
+}
+
+h1 {
+    font-family: "Merriweather", "Times New Roman", serif;
+    color: white;
+    font-size: 56px;
+}
+
+h2 {
+    font-family: "Merriweather", "Times New Roman", serif;
+    color: #5722CD;
+}
+
+h3 {
+    font-family: "Merriweather", "Times New Roman", serif;
+}
+
+h4 {
+    color: white;
+    font-size: 28px;
+    font-weight: lighter;
+}
+
+h5 {
+    color: #5722CD;
+    font-family: "Merriweather", "Times New Roman", serif;
+    font-size: 28px;
+}
+
+nav a:hover {
+    color: #7A7A7A;
+}
+
+nav a:visited {
+    color: white;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-content: space-between;
+    list-style-type: none;
+    font-size: 28px;
+    gap: 50px;
+    margin-right: 100px;
+    padding: 30px;
+}
+
+.home {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 70px;
+    text-align: center;
+    width: 40%;
+    margin: 150px auto 1em;
+}
+
+#squiggleone {
```

Comment:
Je was ontzettend in de buurt met deze! Als je deze jongen aan de onderkant in het midden wil hebben, zul je de position-property moeten gebruiken:

```
#squiggleone {
   position: absolute;
   bottom: 0;
   left: calc(50% - 1em);
}
```

En vergeet niet je header dan een `position: relative;` te geven, anders ben je je squiggle kwijt!